kansha/board/comp.py

```diff
@@ -168,23 +174,26 @@ def copy(self, owner, additional_data):
         additional_data['author'] = owner
 
         cols = [col() for col in self.columns if not col().is_archive]
```

Comment:
Note: when a default archive column will be always created, this `is_archive` stuff will have to be removed